models/intermediate/int_google_play__latest_product_info.sql

```diff
@@ -0,0 +1,38 @@
+with earnings as (
+
+    select *
+    from {{ var('earnings') }}
+
+), transaction_recency as (
+
+    select 
+        package_name,
+        product_title,
+        sku_id,
+        max(transaction_pt_timestamp) as last_transaction_at
+    
+    from earnings
+    group by 1,2,3
+
+), order_product_records as (
+
+    select 
+        *,
+        row_number() over(partition by sku_id order by last_transaction_at desc) as n
+
+    from transaction_recency
+
+), latest_product_record as (
+
+    select 
+        package_name,
+        product_title,
+        sku_id
+
+    from order_product_records
+    where n = 1
```

Comment:
Can you add a bit more color to why we are taking this approach here?

Reply:
yeah, i'll add this as comments as well: in the finance report, i want to bring in the product title that we're reporting on.  however, a product (with its unique `sku_id`) can have its title changed over time. thus, i am taking the title used in the latest transaction involving the product, as the title is stored in there.

Reply:
That's a great call!